client/src/app/template-questionnaire/template-questionnaire.data.ts

```diff
@@ -134,7 +133,7 @@ export class TemplateSpecification {
   private addTypeSpec(ts: TypeSpec, recordInfo: boolean): void {
     //cloning to ensure the source object doesn't get overwritten by merges
     let clone = Object.assign({}, ts);
-    clone.embedProcess = 'category' in clone && clone['category'] == 'biomaterial' ? recordInfo : false;
+    clone.embedProcess = 'category' in clone && clone['schemaName'] !== 'donor_organism' && clone['category'] === 'biomaterial' ? recordInfo : false;
```

Comment:
I modified the failing test for this but i wasn't quite sure how this really works. I looked at the generated spreadsheet and if i haven't overlooked it, i noticed i didn't see any embedded process column inside any worksheets (cell suspension, etc)